### ISSUE TITLE
RN 0.61 React/RCTImageLoader.h file not found

### ISSUE THREAD

**Issue by buraky84** [react-native-maps/react-native-maps#3142]
/node_modules/react-native-maps/lib/ios/AirMaps/AIRMapOverlay.m:5:9: fatal error: 'React/RCTImageLoader.h' file not found
#import <React/RCTImageLoader.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

=====================================

  RCTImageLoader.h is deprecated with RN 0.61. Instead, correct import should be RCTImageLoaderProtocol.h

## Environment info

System:
    OS: macOS 10.15
    CPU: (8) x64 Intel(R) Core(TM) i5-8279U CPU @ 2.40GHz
    Memory: 607.05 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.16.3 - /usr/local/bin/node
    npm: 6.9.0 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.0, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
    Android SDK:
      Android NDK: 20.0.5594570
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5791312
    Xcode: 11.0/11A420a - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.2 => 0.61.2

**Comment by anastely** [comment]
I think there is not support RN 0.61 yet

**Comment by ferryhinardi** [comment]
Any Updates about this?

**Comment by qyhongfan** [comment]
same issue

**Comment by valeriik** [comment]
I have `"react-native": "`**0.59.8**`"` and with fix mentioned above (https://github.com/react-native-community/react-native-maps/pull/3136), unfortunately I got the same error.
Downgrading to previous https://github.com/react-native-community/react-native-maps/commit/cc12beecc71363f68d9beebb00b5885ed6d3912c commit resolve the issue.

So now my _package.json_ looks like here:
`"react-native-maps": "git+https://github.com/react-community/react-native-maps.git#cc12beecc71363f68d9beebb00b5885ed6d3912c"`
and all works fine in RN **0.59.8**

So, it seems you fix it for **0.61** but broke it for at least **0.59.8**

**Comment by deepslam** [comment]
I run into the same issue. 0.61.2

**Comment by conor909** [comment]
I think this is a duplicate of #3094

**Comment by mbrucher** [comment]
I added the patch from #3136 to the latest released version, but I ended up with a blank map. Is there something more to do here?

**Comment by buraky84** [comment]
Still have the same issue with React-native-maps 0.26.1. When do you plan a release?

**Comment by xtealer00** [comment]
**Ran into this issue with react-native 0.61.4**
----
**Fixed Adding these entries to Podfile:**
```
  pod 'React-CoreModules', :path => '../node_modules/react-native/React/CoreModules'
  pod 'FBReactNativeSpec', :path => "../node_modules/react-native/Libraries/FBReactNativeSpec"
  pod 'RCTTypeSafety', :path => "../node_modules/react-native/Libraries/TypeSafety"
  pod 'FBLazyVector', :path => "../node_modules/react-native/Libraries/FBLazyVector"
  pod 'RCTRequired', :path => "../node_modules/react-native/Libraries/RCTRequired"
  pod 'ReactCommon/turbomodule/core', :path => "../node_modules/react-native/ReactCommon"
```
then ran `pod install`
then edited on XCODE the **React.podspec** file (can be found at Pods/Development Pods/React/Pod/React.podspec) and added before the **end** tag the following lines:
```
  s.dependency "React-RCTActionSheet", version
  s.dependency "React-CoreModules", version
```

**Comment by mamta-deswal** [comment]
This issue is because in React-Native 0.61 RCTImageLoader replace with RCTImageLoaderProtocol.
Please update library accordingly. I'm using react-native version 0.61.5 and facing the same issue.

**Comment by JavierT7** [comment]
I run into the same issue. 0.61.8 did you find any solution?

**Comment by JavierT7** [comment]
I run into the same issue using RN v0.61.8, did you find any solution?

**Comment by Naartti** [comment]
I ran into this issue after upgrading RN to 0.61.4, though managed to fix it after adjusting podfile so it matched [React Native Upgrade Helper](https://react-native-community.github.io/upgrade-helper/?from=0.59.9&to=0.61.4) (basically changed to the same podfile as [the template](https://raw.githubusercontent.com/react-native-community/rn-diff-purge/release/0.61.4/RnDiffApp/ios/Podfile)).

In my case, I was missing:
*FBLazyVector, FBReactNativeSpec, RCTRequired, RCTTypeSafety, ReactCommon/jscallinvoker* and *ReactCommon/turbomodule/core*.

I had to change the import path for:
*React-CoreModules, React-Core/DevSupport* and *React-Core/RCTWebSocket*.

Finally I needed to change *yoga* to *Yoga*.

**Comment by wilsolutions** [comment]
Could anyone that got it working with 0.61.5 create a branch or PR?
thank you

**Comment by Nargle30** [comment]
The same with RN v0.60.3, showed up only after updating react-native-maps from v0.26.1 to v0.27.0

Error in Xcode: 
`'React/RCTImageLoaderProtocol.h' file not found`

Error in terminal: 
```
The following build commands failed:
	CompileC /path/ios/build/reactNativeStarter/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/react-native-maps.build/Objects-normal/x86_64/AIRMapOverlay.o /path/node_modules/react-native-maps/lib/ios/AirMaps/AIRMapOverlay.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
```

**Comment by NicoCevallos** [comment]
Hi, I faced the same issue and found a way to solve it.
As the changelog for version 0.27 says:

> ## 0.27.0 (March 4, 2020)
> ...
> * iOS: [#3136](https://github.com/react-native-community/react-native-maps/pull/3136) Use RCTImageLoaderProtocol for RN v0.61.0
> ...

So, you need to manual change `RCTImageLoaderProtocol` to `RCTImageLoader`
But to avoid do that by every dev in the team, you can use [`patch-package`](https://github.com/ds300/patch-package) to create and patch to this issue, so then everyone can apply the patch by `postinstall`

The same solution can be used for users with `react-native-maps` < 0.27.0 and `react-native` >= 0.61

I hope you find it usefull

**Comment by qualitymanifest** [comment]
@NicoCevallos thank you!! I spent hours going through issues trying out different fixes. This is what finally did it. I downgraded react-native-maps to 0.26.1 (last version before the breaking change) and now it works with react-native 0.60.5

Tagging @rborn since you were very active in some of the other issues I saw. Figured it might be good to know for when this pops up again, might be good to add something to the installation instructions too
